pkg/agent/secondarynetwork/podwatch/controller.go

```
@@ -133,6 +134,16 @@ func NewPodController(
 		},
 		resyncPeriod,
 	)
+
+	err = pc.initializeSecondaryInterfaceStore()
+	if err != nil {
+		klog.ErrorS(err, "Failed to initialize the secondary bridge interface store")
```

We should return the error and fail the agent startup.

---

secondary bridge interface store -> secondary interface store